content/php/concepts/math-functions/terms/is-finite/is-finite.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'is_finite()'
+Description: 'Evaluates whether a value is a legal finite number or not.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`is_finite()`** function evaluates whether the value is finite or not and returns a *boolean* value.
+
+## Syntax
+
+```pseudo
+$result = is_finite(value);
+```
+
+Returns `1` (`true`) if the `value` is finite, nothing (`false`) otherwise.
+
+## Example
+
+The example below demonstrates the use of the `is_finite()` function to check whether a number is finite or not.
+
+```php
+<?php
+  $result = is_finite(100);
+  echo $result; // Output: 1
+?>
+```
+
+## Codebyte Example
+
+The codebyte below uses the `is_finite()` function on the values of `100` and `log(0)` to evaluate whether they are finite:
```

Comment:
Part of this example is just duplicating the one above. I would advise updating it so that it contains cases (test values) that have not been used already